deepchecks/nlp/checks/model_evaluation/weak_segments_performance.py

```diff
@@ -105,8 +110,9 @@ def run_logic(self, context: Context, dataset_kind) -> CheckResult:
                                                    dummy_model=dummy_model, scorer=scorer)
 
         if len(weak_segments) == 0:
-            raise DeepchecksProcessError('WeakSegmentsPerformance was unable to train an error model to find weak '
-                                         f'segments. Try increasing n_samples or supply more {self.segment_by}.')
+            display_msg = 'WeakSegmentsPerformance was unable to train an error model to find weak segments.'\
+                          f'Try supplying more {self.segment_by}.'
+            return CheckResult(value={'message': display_msg}, display=[display_msg])
```

Comment:
can have the value directly a str I think

Reply:
Actually the conditions expect the value to be a Dict.